src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -380,15 +380,17 @@ private async Task<IEnumerable<PackageItemViewModel>> LoadNextPageAsync(IPackage
 
         private async Task WaitForCompletionAsync(IItemLoader<PackageItemViewModel> currentLoader, CancellationToken token)
         {
-            var progress = new Progress<IItemLoaderState>(
+            IProgress<IItemLoaderState> progress = new Progress<IItemLoaderState>(
                 s => HandleItemLoaderStateChange(currentLoader, s));
 
             // run to completion
             while (currentLoader.State.LoadingStatus == LoadingStatus.Loading)
             {
                 token.ThrowIfCancellationRequested();
-                await currentLoader.UpdateStateAsync(progress, token);
+                await currentLoader.UpdateStateAsync(progress: null, token);//don't report progress to UI while waiting for packages list
```

Comment:
nit space before/after //

Reply:
Given that CI is green for this PR, I am happy to address this feedback in another PR. Hope you are okay with that.